tests/Supportive/DependencyInjection/DeptracExtensionTest.php

```diff
@@ -32,6 +32,14 @@ final class DeptracExtensionTest extends TestCase
         ],
     ];
 
+    private array $analyserDefaults = [
+        'internal_tag' => '@internal',
+        'types' => [
+            'class',
```

Comment:
Please keep the ENUM values instead of hardcoded text.

Reply:
This is apparently not supported in an initializer literal. PHP was complaining about it. I could make a static method that returns the array, then it would work.

Reply:
Please, it would be less error prone.